parity-clib/src/lib.rs

```diff
@@ -162,3 +163,21 @@ pub extern fn parity_rpc(client: *mut c_void, query: *const char, len: usize, ou
 		}).unwrap_or(1)
 	}
 }
+
+#[no_mangle]
+pub extern fn parity_set_panic_hook(callback: extern "C" fn(*mut c_void, *const c_char, usize), param: *mut c_void) {
```

Comment:
should this not be `unsafe` to invoke?

Reply:
right now I can call this from Rust code with crap data and it will register it as a thread-safe panic hook